doc/sample_azure-monitor.rst

```diff
@@ -9,7 +9,7 @@ Create the Monitor client
 The following code creates an instance of the client.
 
 You will need to provide your ``subscription_id`` which can be retrieved
-from `your subscription list <https://manage.windowsazure.com/#Workspaces/AdminTasks/SubscriptionMapping>`__.
+from `your subscription list <https://ms.portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade>`__.
```

Comment:
Change the host: "ms.portal.azure.com" is because your subscription is internal. Use "portal.azure.com" and check this works please.